backend/chunker/chunker.go

```diff
@@ -2143,8 +2201,9 @@ type metaSimpleJSON struct {
 	Size     *int64 `json:"size"`    // total size of data chunks
 	ChunkNum *int   `json:"nchunks"` // number of data chunks
 	// optional extra fields
-	MD5  string `json:"md5,omitempty"`
-	SHA1 string `json:"sha1,omitempty"`
+	MD5    string `json:"md5,omitempty"`
+	SHA1   string `json:"sha1,omitempty"`
+	XactID string `json:"xactID,omitempty"` // transaction ID for a completed upload
```

Comment:
Please consider using a short json name like "txn" to save 3 bytes and keep metaobject small.

Reply:
Note: I think that the _xactID_ contraction I once used for "transaction ID" is rather unusual (might be confused with "exact") and shall be replaced e.g. by _txnID_ by a future code refactoring.